pyccel/ast/builtins.py

```diff
@@ -1219,32 +1218,10 @@ def print_string(self):
             return LiteralString(f"<class 'numpy.{dtype}{precision}'>")
 
 #==============================================================================
-python_builtin_datatypes_dict = {
-    'bool'   : PythonBool,
-    'float'  : PythonFloat,
-    'int'    : PythonInt,
-    'complex': PythonComplex,
-    'str'    : LiteralString
-}
-
-def python_builtin_datatype(name):
-    """
-    Given a symbol name, return the corresponding datatype.
-
-    name: str
-        Datatype as written in Python.
-
-    """
-    if not isinstance(name, str):
-        raise TypeError('name must be a string')
-
-    if name in python_builtin_datatypes_dict:
-        return python_builtin_datatypes_dict[name]
-
-    return None
```

Comment:
Can you please explain what happened here? I was just recently reading this part.

Reply:
I don't think it is useful for us to differentiate between datatypes and functions. The two dictionaries point at the same objects and Python itself doesn't differentiate between the two. So I moved anything that was missing into `buitin_functions` and removed this duplication.

Good job you were checking the PRs to notice it changed ;)

Reply:
@EmilyBourne Can you please add a point to the PR description about these changes?

Reply:
I mean, there is a point but it does not elaborate...

Reply:
@yguclu Done